integ-tests/src/cucumber/java/configuration/CucumberConfiguration.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package configuration;
+
+import com.amazonaws.blox.dataserviceclient.v1.client.DataServiceLambdaClientConfiguration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import wrappers.DataServiceWrapper;
+
+@Configuration
+@Import(DataServiceLambdaClientConfiguration.class)
+@ComponentScan("steps")
+public class CucumberConfiguration {
+
+  @Autowired private DataServiceLambdaClientConfiguration dataServiceLambdaClient;
+
+  @Bean
+  public DataServiceWrapper dataServiceWrapper() {
+    return new DataServiceWrapper(dataServiceLambdaClient.dataService());
```

Comment:
You could avoid the explicit autowire by declaring a `DataService` parameter for the `dataServiceWrapper` bean definition method.

Reply:
you mean dataservicewrapper(Dataservice dataservice) will autowire the bean?